strapi/strapi
# chore: remove default index page from public middleware

### What does it do?

Remove the default index page and redirect to the admin directly

### Why is it needed?

To make it easier to find the admin and remove old code.

### How to test it?

go to http://localhost:1337/ and get redirect to /admin. Check uploaded files are still accessible and visible in the media lib

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
